library/lib/edges/EdgeProps.ts

```diff
@@ -0,0 +1,15 @@
+import { Edge, EdgeProps } from "@xyflow/react"
+
+export type CustomEdgeProps = {
+  sourceRole: string | null
+  sourceMultiplicity: string | null
+  targetRole: string | null
+  targetMultiplicity: string | null
+}
```

Comment:
can we do like this and remove all 

```suggestion
export type CustomEdgeProps = {
  sourceRole?: string 
  sourceMultiplicity?: string 
  targetRole?: string 
  targetMultiplicity: string
}
```

and then remove all in initialElements
```
    data: {
      sourceRole: null,
      sourceMultiplicity: null,
      targetRole: null,
      targetMultiplicity: null,
    },
```